File: day14/AsyncDemo.API/Service/UserService.cs
public class UserService: IUserService
{
    public async Task<List<User>> GetUsersAsync()
    {
        await Task.Delay(3000);
        return new List<User>
        {
            new User(1, "Vishal"),
            new User(2, "Rahul"),
            new User(3, "Vishal2")
        };
    }
}

File: day28/interpolationApp/src/app/services/userService.ts
import { Injectable } from '@angular/core';

@Injectable({
  providedIn: 'root',
})
export class UserService {
  users:string[] = ['Vishal', 'Rahul', 'Amit'];
  getUsers(){
    return this.users;
  }
  addUser(user:string){
    this.users.push(user);
  }

}
